ibm/acctest/acctest.go

```diff
@@ -877,6 +880,11 @@ func init() {
 	if ClusterName == "" {
 		fmt.Println("[INFO] Set the environment variable IBM_CONTAINER_CLUSTER_NAME for ibm_container_nlb_dns resource or datasource else tests will fail if this is not set correctly")
 	}
+
+	DedicatedHostPoolID = os.Getenv("IBM_CONTAINER_DEDICATEDHOST_POOL_ID")
+	if ClusterName == "" {
```

Comment:
```suggestion
	if DedicatedHostPoolID == "" {
```

I also wonder whether this parameter needs to be optional. Would this mean that all tests need to specify a DedicatedHostPoolID?

Reply:
The argument is optional by default, fixed the problems with this